### DISCUSSION TITLE
SnowDDL Single DB query

### DISCUSSION THREAD

**Discussion by chintan-episource** [littleK0i/SnowDDL#19] [Q&A] [answered]
Hello- Using SingleDB feature, i have multiple schemas in a DB. Now, for one schema , i want to exclude object type -viz. Stage. and for other schema i use create stage feature. If there possibility to exclude based on object name or exclude object types for a schema ?

**Comment by littleK0i** [comment, marked as answer]
At this moment the general advice is to avoid mixing object types managed by SnowDDL and managed by other means within one schema. It would complicate things too much.

You may consider the following approach:

```
schema1 (managed entirely by SnowDDL)
|-- tables
|-- views
|-- stages

schema2 (created by SnowDDL, but objects are managed by something else)
|-- stages (created externally)
```

In order to prevent SnowDDL from dropping stages created by other tools in specific schema, you may set `is_sandbox: true` option in `params.yaml` for this schema. Relevant documentation page: https://docs.snowddl.com/basic/yaml-configs/schema

**Reply by chintan-episource** [reply]
I did put params.yaml with these settings in schema having objects not create by snowddl , still it showed same issue . it shows objects to be dropped during plan command.

**Reply by littleK0i** [reply]
Could you attach a zip with the current config? Most likely misconfiguration, I'll take a look.

Here is an example of `params.yaml` with `is_sandbox: true` flag: https://github.com/littleK0i/SnowDDL/blob/451083510209935e64f1e05e24e0a75eb6ce34ac/snowddl/_config/sample01_01/snowddl_db/sakila_sandbox/params.yaml

**Reply by chintan-episource** [reply]
Sorry, there was a typo. Apologies. Thanks for your help.
